driver/src/main/scala/core/nodeset.scala

```diff
@@ -667,6 +655,53 @@ final class ChannelFactory private[reactivemongo] (
     pipeline
   }
 
+  private def sslContext = {
+    import java.io.FileInputStream
+    import java.security.KeyStore
+    import javax.net.ssl.{ KeyManagerFactory, TrustManager }
+
+    val keyManagers = Option(System.getProperty("javax.net.ssl.keyStore")).map { path =>
+
+      val password = Option(System.getProperty("javax.net.ssl.keyStorePassword")).getOrElse("")
```

Comment:
Any chance of a response? Sorry to be pushy, it's just you've been so quick to respond so far so thought you may have missed my question. Would be fantastic to get this merged, I'm going to start looking at the integration testing this evening.